### PR TITLE
Updating Demo Notebooks for How-to-Guides

### PR DESCRIPTION
**Pull request by shir22**
also deleted a few redundant old notebooks

### CODE REVIEW

File: notebooks/examples/create_a_custom_suite.ipynb [hunk 1]
```diff
@@ -0,0 +1,759 @@
+{
```
Comment (matanper) [inline notebooks/examples/create_a_custom_suite.ipynb:1]: <p>Is that the place to mention the option of "start with predefined suite and slightly modify it to your needs"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (matanper) [inline notebooks/examples/create_a_custom_suite.ipynb:1]: <p>I don't think this is something we want to encourage to do... better to just directly refer to the API ref</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (matanper) [inline notebooks/examples/create_a_custom_suite.ipynb:1]: <p>I think that it's better to put the imports with the code for copy &amp; paste simplicity.</p><p>also I'd add after the definition a print of the suite</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: notebooks/examples/quickstart_in_5_minutes.ipynb [hunk 2]
```diff
@@ -0,0 +1,2112 @@
+{
```
Comment (matanper) [inline notebooks/examples/quickstart_in_5_minutes.ipynb:1]: <p>"This notebook" -&gt; "This guide/documentation" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (matanper) [inline notebooks/examples/quickstart_in_5_minutes.ipynb:1]: <p>all this 4 blocks of code preparations are mostly noise, if possible I'd put them in a single cell, even a collapsed one if possible, as I don't think most users will care to see this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: notebooks/examples/create_a_custom_suite.ipynb (hunk 1 above)
Comment (shir22) [reply notebooks/examples/create_a_custom_suite.ipynb:1]: <p>Can we remove &amp; add checks to built-in suites?</p><p>If so I think we should add that, but also a part that describes how to do that.</p>

Comment (shir22) [reply notebooks/examples/create_a_custom_suite.ipynb:1]: <p>Yes this was written before we had it. and then I thought it's kind of convenient.</p><p>but will delete :)</p>

Comment (matanper) [reply notebooks/examples/create_a_custom_suite.ipynb:1]: <p>yes it's possible</p>

Comment (shir22) [reply notebooks/examples/create_a_custom_suite.ipynb:1]: <p>Ok so I'll add a short description, with a todo for the example</p><p> </p>

**Review by ItayGabbay — APPROVED** [review]
Looks good to me. I wish we would solve the autocomplete part. (Maybe recommend users to use JupyterLab?)

**Pull request merged**